util/extsort/external_sorter_test.go

```diff
@@ -42,19 +36,9 @@ func runCommonTest(t *testing.T, sorter extsort.ExternalSorter) {
 	require.NoError(t, err)
 
 	rng := rand.New(rand.NewSource(0))
```

Comment:
recommend that we use time.now as seed and t.Log it to help debug the failed case